### DISCUSSION TITLE
Future of this project

### DISCUSSION THREAD

**Discussion by maxdanilov** [actions/actions-runner-controller#2751] [Questions] [answered]
Hi, 

I'd like to ask about the plans for the long-term future of this project.

I'm observing certain struggles within the project recently, e.g., the last patch release happened over 2 months ago, or the stability PR submissions from the open-source contributors don't seem to be getting reviews ([example](https://github.com/actions/actions-runner-controller/pull/2671)). 
On the other head, this repository has been moved to GitHub's governance (the [github.com/actions](https://github.com/actions) org), and GitHub has been featuring this project as General Availability in its [blog](https://github.blog/changelog/2023-06-30-github-actions-actions-runner-general-availability/)

That makes me wonder: how alive is the project and how it's going to move forward: keep being the open-source driven project or go in the hands of GitHub's full-time engineers for further development?

**Comment by maxdanilov** [comment, marked as answer]
My questions were answered here: https://github.com/actions/actions-runner-controller/discussions/2775

**Comment by AnnaDzheeeennn** [comment]
***@***.***

Ср, 26 июля 2023 г. в 9:27 PM, Maxim Danilov ***@***.***>:
